components/home/index.js

```diff
@@ -34,27 +36,37 @@ class Home extends Component {
         longitudeDelta: 0.05
       }
     });
-  }
+  };
 
-  // Function : change the visiblity of the switchCampuses component
-  // parameter: boolean to set the visibility (false: unvisible)
-  changeVisibilityTo = (visibility) => {
-    this.setState({ isVisible: visibility });
+  changeVisibilityTo = boolean => {
+    this.setState({ isVisible: boolean });
+  };
+  changeVisibilityToSearch = boolean => {
+    this.setState({ isVisible2: boolean})
```

Comment:
instead of calling it isVisible2, why not have isSearchVisibility. This style of naming convention is a little confusing